Bluepill-runner/Bluepill-runner/BPPacker.m

```diff
@@ -85,4 +88,60 @@ + (NSMutableArray *)packTests:(NSArray *)xcTestFiles
     return bundles;
 }
 
+#pragma mark - Private helper methods
+
++ (NSMutableSet *)expandedTests:(NSArray *)configTestCases fromTestFile:(BPXCTestFile *)xctFile {
```

Comment:
`expandTest` not `expandedTests`, like the function above, we use `packTests` not `packedTests` 

also, could you move this function under `normalizedBPConfiguration ` function

And, please and some comments for this function as well, it can be as simple as one line // xxxx

sorry for being picky, good to go after this!